sdn_tests/pins_ondatra/infra_deps.bzl

```diff
@@ -0,0 +1,337 @@
+    )
+
+    go_repository(
+        name = "in_gopkg_yaml_v2",
+        importpath = "gopkg.in/yaml.v2",
+        repo_mapping = repo_map,
+        sum = "h1:D8xgwECY7CYvx+Y2n4sBz93Jn9JRvxdiyyo8CTfuKaY=",
+        version = "v2.4.0",
+    )
+
+    go_repository(
+        name = "io_opencensus_go",
+        importpath = "go.opencensus.io",
+        repo_mapping = repo_map,
+        sum = "h1:y73uSU6J157QMP2kn2r30vwW1A2W2WFwSCGnAVxeaD0=",
+        version = "v0.24.0",
+    )
+
+    go_repository(
+        name = "org_golang_google_grpc",
+        importpath = "google.golang.org/grpc",
+        repo_mapping = repo_map,
+        sum = "h1:EhTqbhiYeixwWQtAEZAxmV9MGqcjEU2mFx52xCzNyag=",
+        version = "v1.54.0",
+    )
+
+    go_repository(
+        name = "org_golang_google_grpc_cmd_protoc_gen_go_grpc",
+        importpath = "google.golang.org/grpc/cmd/protoc-gen-go-grpc",
+        repo_mapping = repo_map,
+        sum = "h1:M1YKkFIboKNieVO5DLUEVzQfGwJD30Nv2jfUgzb5UcE=",
+        version = "v1.1.0",
+    )
+
+    go_repository(
+        name = "org_golang_google_protobuf",
+        importpath = "google.golang.org/protobuf",
+        repo_mapping = repo_map,
+        sum = "h1:kPPoIgf3TsEvrm0PFe15JQ+570QVxYzEvvHqChK+cng=",
+        version = "v1.30.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_exp",
+        importpath = "golang.org/x/exp",
+        commit = "aacd6d4b4611949ff7dcca7a0118e9312168a5f8",
+    )
+
+    go_repository(
+        name = "org_golang_x_net",
+        importpath = "golang.org/x/net",
+        repo_mapping = repo_map,
+        sum = "h1:aWJ/m6xSmxWBx+V0XRHTlrYrPG56jKsLdTFmsSsCzOM=",
+        version = "v0.9.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_sync",
+        importpath = "golang.org/x/sync",
+        repo_mapping = repo_map,
+        tag = "v0.3.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_sys",
+        importpath = "golang.org/x/sys",
+        repo_mapping = repo_map,
+        sum = "h1:3jlCCIQZPdOYu1h8BkNvLz8Kgwtae2cagcG/VamtZRU=",
+        version = "v0.7.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_text",
+        importpath = "golang.org/x/text",
+        repo_mapping = repo_map,
+        sum = "h1:2sjJmO8cDvYveuX97RDLsxlyUxLl+GHoLxBiRdHllBE=",
+        version = "v0.9.0",
+    )
+
+
+    git_repository(
+        name = "com_google_googleapis",
+        remote = "https://github.com/googleapis/googleapis",
+        commit = "c4915db59896a1da45b55507ece2ebc1d53ef6f5",
+        shallow_since = "1642638275 -0800",
+    )
+
+    go_repository(
+        name = "com_github_jstemmer_go_junit_report_v2",
+        importpath = "github.com/jstemmer/go-junit-report/v2",
+        sum = "h1:BVBb1o0TfOuRCMykVAYJ1r2yoZ+ByE0f19QNF4ngQ0M=",
+        version = "v2.0.1-0.20220823220451-7b10b4285462",
+    )
+
+    go_repository(
+        name = "com_github_patrickmn_go_cache",
+        importpath = "github.com/patrickmn/go-cache",
```

Comment:
@vamsipunati, @parth-opensrc 
This and the previous one, are we ok to include these as dependencies in sonic-net ?

Reply:
Ondatra needs this to be built.
https://github.com/openconfig/ondatra/blob/main/internal/ixgnmi/ixgnmi.go#L39, and 
https://github.com/openconfig/ondatra/blob/main/report/report_test.go#L23

We are using checksum, so it should be fine, I guess.

Reply:
@parth-opensrc Tried removing the rules, The build was successful. Attaching the build info in the chat box 

       INFO: Analyzed 0 targets (1 packages loaded, 0 targets configured).
       INFO: Found 0 targets...
       INFO: Elapsed time: 0.593s, Critical Path: 0.00s
       INFO: 1 process: 1 internal.
       INFO: Build completed successfully, 1 total action